CefSharp/ModelBinding/DefaultBinder.cs

```diff
@@ -55,13 +55,13 @@ public virtual object Bind(object obj, Type modelType)
             if (modelType.IsCollectionOrArray())
             {
                 //make sure it has a generic type
-                if (modelType.GetTypeInfo().IsGenericType)
+                if (modelType.IsGenericType)
                 {
                     genericType = modelType.GetGenericArguments().FirstOrDefault();
                 }
                 else
                 {
-                    var ienumerable = modelType.GetInterfaces().Where(i => i.GetTypeInfo().IsGenericType).FirstOrDefault(i => i.GetGenericTypeDefinition() == typeof(IEnumerable<>));
+                    var ienumerable = modelType.GetInterfaces().Where(i => i.IsGenericType).FirstOrDefault(i => i.GetGenericTypeDefinition() == typeof(IEnumerable<>));
```

Comment:
It looks like the difference of getTypeInfo is that it loads the type into the assembly.  I don't think it matters too much, since nothings broken.  But it makes me slightly nervous.